drivers/power/pm_activity.c

```diff
@@ -143,21 +182,40 @@ void pm_stay(int domain, enum pm_state_e state)
 
 void pm_relax(int domain, enum pm_state_e state)
 {
-  FAR struct pm_domain_s *pdom;
-  irqstate_t flags;
+  DEBUGASSERT(state < PM_COUNT);
+  DEBUGASSERT(domain >= 0 && domain < CONFIG_PM_NDOMAINS);
 
-  /* Get a convenience pointer to minimize all of the indexing */
+  pm_wakelock_relax(&g_wakelock[domain][state]);
+}
 
-  DEBUGASSERT(domain >= 0 && domain < CONFIG_PM_NDOMAINS);
-  pdom = &g_pmglobals.domain[domain];
+/****************************************************************************
+ * Name: pm_staytimeout
+ *
+ * Description:
+ *   This function is called by a device driver to indicate that it is
+ *   performing meaningful activities (non-idle), needs the power at kept
+ *   last the specified level.
+ *   And this will be timeout after time (ms), menas auto pm_relax
+ *
+ * Input Parameters:
+ *   domain - The domain of the PM activity
+ *   state - The state want to stay.
```

Comment:
```suggestion
 *   state   - The state want to stay.
```
Please line this up

Reply:
don't need

Reply:
see https://github.com/apache/incubator-nuttx/blob/master/sched/sched/sched_get_stackinfo.c#L43-L46

Reply:
done

Reply:
@davids5 please review